### ISSUE TITLE
训练时报了AttributeError: 'Namespace' object has no attribute 'image_weights'的错误，

### ISSUE THREAD

**Issue by OU1005465816** [ultralytics/yolov5#9481]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### YOLOv5 Component

Training

### Bug

截图上传不了

### Environment

yolov5 cpu windows

### Minimal Reproducible Example

最终错误定位到这段代码里
train_loader, dataset = create_dataloader(train_path, imgsz, batch_size // WORLD_SIZE, gs, single_cls,
                                              hyp=hyp, augment=True, cache=None if opt.cache == 'val' else opt.cache,
                                              rect=opt.rect, rank=LOCAL_RANK, workers=workers,
                                              image_weights=opt.image_weights, quad=opt.quad,
                                              prefix=colorstr('train: '), shuffle=True)

### Additional

无 

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@OU1005465816 --image-weights works correctly:

<img width="1293" alt="Screenshot 2022-09-19 at 11 58 05" src="https://user-images.githubusercontent.com/26833433/190993593-58e32735-891e-41b7-a4c7-6a7f69041eee.png">

We've created a few short guidelines below to help users provide what we need in order to start investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us better understand and diagnose your problem. 

Thank you! 😃

**Comment by BreezeHavana** [comment]
there is no image_weights but images_weights in namespace, so change it.

**Comment by glenn-jocher** [comment]
@BreezeHavana thank you for notifying us of this spelling error. We appreciate your input and apologize for any confusion it may have caused to users. Our team has fixed the spelling error and updated the spelling to `--image-weights` in the latest version of our YOLOv5 repo. If you encounter any further issues or have any suggestions, please do not hesitate to let us know. We are committed to providing the best possible user experience and appreciate your valuable feedback.
